### ISSUE TITLE
Trying to print binary file in AUR package

### ISSUE THREAD

**Issue by G0rger** [lxgr-linux/buildaur#5]
When an AUR package contains a binary file (like a compressed tar archive) it fails with:
```
Traceback (most recent call last):
  File "/usr/bin/buildaur", line 579, in <module>
    main()
  File "/usr/bin/buildaur", line 448, in main
    install(pkgs)
  File "/usr/bin/buildaur", line 299, in install
    print("\033[37m"+str(open(file, "rt").read())+"\033[0m", end="")
  File "/usr/lib/python3.10/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x8b in position 1: invalid start byte
```

**Comment by lxgr-linux** [comment]
This was fixed in 78931f2, but is not in the newest version yet.

**Comment by lxgr-linux** [comment]
> This was fixed in [78931f2](https://github.com/lxgr-linux/buildaur/commit/78931f21cc1862a6f04e4668b172aaaccedeb566), but is not in the newest version yet.

The newest version is now in the AUR.
